Composer/packages/client/src/pages/publish/notifications.tsx

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+/** @jsx jsx */
+import { jsx, css } from '@emotion/core';
+import formatMessage from 'format-message';
+import { ProgressIndicator } from 'office-ui-fabric-react/lib/ProgressIndicator';
+import { Icon } from 'office-ui-fabric-react/lib/Icon';
+
+import { CardProps } from '../../components/Notifications/NotificationCard';
+
+import { IBotStatus } from './botStatusList';
+
+const cardContent = css`
+  display: flex;
+  padding: 0 16px 16px 12px;
+  min-height: 64px;
+  align-items: center;
+`;
+
+const cardDetail = css`
+  margin-left: 8px;
+  flex-grow: 1;
+`;
+
+const infoType = css`
+  margin-top: 4px;
+  color: #0078d4;
+`;
+
+const cardDescription = css`
+  text-size-adjust: none;
+  font-size: 12px;
+  line-height: 16px;
+  margin-right: 16px;
+  word-break: break-word;
+`;
+
+export const publishedNotificationCard = (item: IBotStatus): CardProps => {
```

Comment:
this is a function that returns a value, better naming would be getPublishedNotificationCardProps